htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -1398,7 +1398,7 @@ public function create($user, $notrigger = 0)
 		$sql .= ", '".$this->db->escape($this->ref_supplier)."'";
 		$sql .= ", '".$this->db->escape($this->note_private)."'";
 		$sql .= ", '".$this->db->escape($this->note_public)."'";
-		$sql .= ", ".((int) $conf->entity);
+		$sql .= ", ".setEntity($this);
```

Comment:
You must use IN (".getEntities('tablename').") to manage multicompanies in the where